src/demos/_assets/demoPageReloader.ts

```diff
@@ -3,7 +3,7 @@
   const getSocketUrl = ({ hostname, port, protocol }: Location): string =>
     protocol === "https:" ? "wss:" : "ws:" + "//" + hostname + ":" + port + "/";
 
-  const clientWebSocket = new window.WebSocket(getSocketUrl(window.location), ["xmpp"]);
+  const clientWebSocket = new (window as any).WebSocket(getSocketUrl(window.location), ["xmpp"]);
```

Comment:
Do you need the `any` type cast here? `WebSocket` should be available. Did a quick test (`npm run build` && `npm start`) and it works fine.

Reply:
I'll revert. might have had something to do with some typings